### PR TITLE
Cutmix -> CutMix

### PR DESCRIPTION
**Pull request by NicolasHug**
@pmeier @vfdev-5: discuss

Follow up to https://github.com/pytorch/vision/pull/7772#discussion_r1278955408

(Note that timm names them `Cutmix` and `Mixup`)

### CODE REVIEW

**Comment by vfdev-5** [comment]
@NicolasHug My comment was only about the documentation, class names can be Cutmix and Mixup

File: torchvision/prototype/transforms/_augment.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from torchvision.transforms.v2.utils import has_any, is_simple_tensor, query_size
 
 
-class _BaseMixupCutmix(_RandomApplyTransform):
+class _BaseMixUpCutMix(_RandomApplyTransform):
```
Comment (pmeier) [inline torchvision/prototype/transforms/_augment.py:17]: Can't we delete them from prototype now that we have a proper implementation in v2? If you do, note that there are tests for this in `test/test_prototype_transforms.py`

**Review by pmeier — APPROVED** [review]
LGTM, thanks Nicolas

File: torchvision/prototype/transforms/_augment.py (hunk 1 above)
Comment (NicolasHug) [reply torchvision/prototype/transforms/_augment.py:17]: will do in another PR

**Pull request merged**